.gitignore

```diff
@@ -0,0 +1 @@
+.bash_profile
```

Comment:
Why did you have to ignore the .bash_profile? That file should be in your user root, and if you're seeing that in your repo then you either did an `init` on the user root (which is tracking this whole project), or you created a .bash_profile in this folder, and can move it to the root dir.